index.html

```diff
@@ -285,6 +285,7 @@ <h2 class="h2-desktop">Let’s chat!</h2>
         </p>
       </div>
       <form
+        novalidate
         action="https://formspree.io/f/mlezyjrr"
         method="post"
         onsubmit="return ErrorMessage()"
```

Comment:
- [ ] The Error message should be displayed close to the submit button, kindly implement this